### ISSUE TITLE
native-image --version should include OpenJDK build number

### ISSUE THREAD

**Issue by jerboaa** [graalvm/mandrel#68]
Native image currently doesn't print the build number. That's an important detail for issue diagnosis, especially when we are in the early access phase.

```
$ native-image --version
GraalVM Version 20.1.0 (Java Version 11.0.8-ea)
```

Consider using something like this to retrieve the OpenJDK version at build time (to be then passed as org.graalvm.version propery for the mandrel build):

```
$ cat PrintVersion.java
import java.lang.Runtime.Version;

public class PrintVersion {

    public static void main(String[] args) {
        Version v = Runtime.version();
        int build = 0;
        if (v.build().isPresent()) {
            build = v.build().get();
        }
        String optional = "";
        if (v.pre().isPresent()) {
            optional = "-" + v.pre().get();
        }
        System.out.format("%s.%s.%s%s+%s\n", v.feature(), v.interim(), v.update(), optional, build);
    }

}
$ java PrintVersion.java
11.0.8-ea+5
```

**Comment by zakkak** [comment]
The Java version is actually set here https://github.com/graalvm/mandrel/blob/be81deca5e4eab96725fba3ba2a2710008908e0b/substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java#L85 (not through `org.graalvm.version`).

If this is something needed only for the ea phase I suggest we patch it in mandrel/20.1, otherwise we should better try to push the change upstream.

Note that it's relatively easy to get this info by running `/path/to/mandrel/java --version` (or just `java --version` in our images)

**Comment by jerboaa** [comment]
+1 on getting this into upstream.

**Comment by zakkak** [comment]
Opened https://github.com/oracle/graal/pull/2614

**Comment by jerboaa** [comment]
oracle#2614 is now resolved. Removing pending upstream label.

**Comment by zakkak** [comment]
This is now merged in 20.1. Pending backport to 20.2
